shard/Dockerfile

```diff
@@ -10,6 +12,9 @@ WORKDIR /app
 
 RUN apt-get -y update
 RUN apt-get -y install python3-pip ffmpeg
+RUN apt-get -y install imagemagick texlive texlive-latex-extra
+
+RUN mv /etc/ImageMagick-6/policy.xml /etc/ImageMagick-6/policy.xml.off
```

Comment:
I would rather not use imagemagick if we can help it.  idk if using a different latex compiler and then using PIL would be a great deal more trouble